### PR TITLE
teleport warning followup

### LINKED ISSUE #8527: Teleport warning followups
ref and after: 
- https://github.com/kodadot/nft-gallery/pull/8505

## Design 
- updated in the teleport v2 page: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=4168-6846&mode=design&t=6fgjdAvQonLDzyHr-11

image

learn more article link: https://hello.kodadot.xyz/multi-chain/existential-deposit

### Warning 
image
- tooltip when hovering over the link "why?" 

### Modal
- if you still decide to proceed, this modal will be displayed 

image
- the continue anyway button is disabled until the checkbox is checked 

dark mode added as well: 
image

### LINKED ISSUE #8548: bad button label in telport
image

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [x] Feature

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #8527
- [x] Closes #8548

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)

## Screenshot 📸

- [x] My fix has changed UI

image

image

image

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at e454f94</samp>

This pull request enhances the user experience and safety of the teleport feature by adding a warning modal and a tooltip to inform the user about the existential deposit risk. It also fixes a minor bug with the currency value display. The changes affect the `Teleport.vue` and `EDWarningModal.vue` components and the `en.json` localization file.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at e454f94</samp>

> _To teleport funds, you must know_
> _The existential deposit, though_
> _If you send too few_
> _A modal will show_
> _And warn you before you can go_

### CODE REVIEW

**Comment by exezbcz** [comment]
can we maybe add "DOT" and also "(existential deposit)" so its clear why 

image

File: components/teleport/Teleport.vue [hunk 1]
```diff
@@ -139,6 +163,16 @@
       {{ $t('teleport.ownerMessage') }}
     </div>
   </form>
+  <EDWarningModal
```
Comment (roiLeo) [inline components/teleport/Teleport.vue:166]: auto imported
```suggestion
  <TeleportEdWarningModal
```

File: components/teleport/Teleport.vue [hunk 2]
```diff
@@ -155,13 +189,19 @@ import formatBalance from '@/utils/format/balance'
...
+  NeoField,
+  NeoIcon,
+  NeoInput,
+  NeoTooltip,
+} from '@kodadot1/brick'
 import { blockExplorerOf } from '@/utils/config/chain.config'
 import { simpleDivision } from '@/utils/balance'
 import { useFiatStore } from '@/stores/fiat'
 import { existentialDeposit } from '@kodadot1/static'
+import EDWarningModal from './EDWarningModal.vue'
```
Comment (roiLeo) [inline components/teleport/Teleport.vue:204]:
```suggestion
```

**Review by roiLeo — APPROVED** [review]

**Comment by prury** [comment]
- Cursor style when hovering over Funds at risk message

- mobile: 
image

**Comment by daiagi** [comment]
> * Cursor style when hovering over Funds at risk message

fixed - although it is same as all text in the app


>     * mobile:

i whipped up some solution. @exezbcz wdyt?

image

File: components/teleport/Teleport.vue (hunk 1 above)
Comment (daiagi) [reply components/teleport/Teleport.vue]: done

File: components/teleport/Teleport.vue (hunk 2 above)
Comment (daiagi) [reply components/teleport/Teleport.vue]: done

**Comment by prury** [comment]
> > * Cursor style when hovering over Funds at risk message
> 
> fixed - although it is same as all text in the app

forgive me, I've failed to specify that it was only on `why?` and it was pointer style, but since its minor let's go.

**Comment by exezbcz** [comment]
> i whipped up some solution. @exezbcz wdyt?

works for me thanks!

**Comment by daiagi** [comment]
>forgive me, I've failed to specify that it was only on why? and it was pointer style, but since its minor let's go.

it isn't clickable. only hoverable

**Comment by prury** [comment]
> >forgive me, I've failed to specify that it was only on why? and it was pointer style, but since its minor let's go.
> 
> it isn't clickable. only hoverable

True

**Comment by daiagi** [comment]
@exezbcz 

image

**Comment by prury** [comment]
@daiagi strange, i'm still not seeing the currency, even after cache clear or browser change
and now message appears even if i funds on destination chain

image

**Comment by exezbcz** [comment]
Can we push this one ASAP, please as more people are losing their funds

**Comment by yangwao** [comment]
> i'm still not seeing the currency, even after cache clear or browser change
> and now message appears even if i funds on destination chain

@prury let's do follow up, merging it for now, hope it improves over time

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 7.16 USD/DOT ~ 6.983 $DOT
🧗 [1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8](https://polkadot.subscan.io/account/1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)
🔗 [0x229cd8a10d454d22fc8d32c87be6eaa05997ce68df3fac5b1618805569c7dede](https://polkadot.subscan.io/extrinsic/0x229cd8a10d454d22fc8d32c87be6eaa05997ce68df3fac5b1618805569c7dede)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**